packages/edge-config/src/index.common.test.ts

```diff
@@ -78,7 +78,10 @@ describe('when running without lambda layer or via edge function', () => {
         expect(fetchMock).toHaveBeenCalledWith(
           `${modifiedBaseUrl}/item/foo?version=1`,
           {
-            headers: new Headers({ Authorization: 'Bearer token-2' }),
+            headers: new Headers({
+              Authorization: 'Bearer token-2',
+              'x-node-env': 'test',
```

Comment:
Is x-node-env something we use anywhere else at vercel? Otherwise I’d maybe name this x-edge-config-node-env

Reply:
Not that I'm aware of, happy to change